sugoi-api-rest-services/src/main/java/fr/insee/sugoi/services/controller/AppManagedUserAttributeController.java

```diff
@@ -0,0 +1,285 @@
+    if (permissionService.isWriter(realm, storage)) {
+      userService.addAppManagedAttribute(realm, storage, id, attribute);
+      return ResponseEntity.status(HttpStatus.OK).body(userService.findById(realm, storage, id));
+    } else {
+
+      List<String> regexpAttributesAllowed =
+          permissionService.getAllowedAttributePattern(
+              realm,
+              storage,
+              realmProvider
+                  .load(realm)
+                  .getProperties()
+                  .get(GlobalKeysConfig.APP_MANAGED_ATTRIBUTE_PATTERN));
+      // Check if attribute match with allowed pattern
+      for (String regexpAttributeAllowed : regexpAttributesAllowed) {
+        if (attribute.toUpperCase().matches(regexpAttributeAllowed)) {
+          userService.addAppManagedAttribute(realm, storage, id, attribute);
+          return ResponseEntity.status(HttpStatus.OK)
+              .body(userService.findById(realm, storage, id));
+        }
+      }
+      // If no match found then app cannot managed attribute or attribute doesn't math
+      // with allowed pattern
+      throw new AppCannotManagedAttributeException(
+          "Cannot add attribute to user: attribute doesn't match with pattern");
+    }
+  }
+
+  @DeleteMapping(
+      value = {"/realms/{realm}/storages/{storage}/users/{id}/app-managed-attribute"},
+      consumes = {MediaType.APPLICATION_JSON_VALUE},
+      produces = {MediaType.APPLICATION_JSON_VALUE})
+  @Operation(summary = "add attribute managed by application")
+  @PreAuthorize("@NewAuthorizeMethodDecider.isReader(#realm,#storage)")
+  @ApiResponses(
+      value = {
+        @ApiResponse(
+            responseCode = "200",
+            description = "User updated",
+            content = {
+              @Content(
+                  mediaType = "application/json",
+                  schema = @Schema(implementation = User.class))
+            }),
+        @ApiResponse(
+            responseCode = "404",
+            description = "User not found",
+            content = {@Content(mediaType = "application/json")}),
+        @ApiResponse(
+            responseCode = "400",
+            description = "Invalid combinaison of id between body and path",
+            content = {@Content(mediaType = "application/json")})
+      })
+  public ResponseEntity<?> deleteUserAttributesManagedByApp(
+      @Parameter(
+              description = "Name of the realm where the operation will be made",
+              required = true)
+          @PathVariable("realm")
+          String realm,
+      @Parameter(
+              description = "Name of the userstorage where the operation will be made",
+              required = true)
+          @PathVariable("storage")
+          String storage,
+      @Parameter(description = "User's id to update", required = true) @PathVariable("id")
+          String id,
+      @Parameter(description = "value of attribute to delete", required = true) @RequestBody
+          String attribute) {
```

Comment:
This shoulkd not be a request body, either in path or query string : 
/app-managed-attribute/$(attribute) or app-managed-attribute?value=$value